src/o365/teams/commands.ts

```diff
@@ -10,8 +10,9 @@ export default {
   TEAMS_GUESTSETTINGS_SET: `${prefix} guestsettings set`,
   TEAMS_CHANNEL_ADD: `${prefix} channel add`,
   TEAMS_CHANNEL_GET: `${prefix} channel get`,
-  TEAMS_CHANNEL_LIST:`${prefix} channel list`,
+  TEAMS_CHANNEL_LIST: `${prefix} channel list`,
   TEAMS_CHANNEL_SET: `${prefix} channel set`,
+  TEAMS_CHANNEL_REMOVE: `${prefix} channel remove`,
```

Comment:
Could you please move the remove above set? We try to keep them alphabetically , the easiest to guess the order is to look the the VS Code file explorer where they are ordered A-Z.
![image](https://user-images.githubusercontent.com/1049476/64358496-79959600-cffe-11e9-9796-0b8e7afa9abb.png)